### DISCUSSION TITLE
Support Summer '15 Batch REST API

### DISCUSSION THREAD

**Discussion by sfdeveloperhub** [jsforce/jsforce#1514] [Ideas]
Unless I am missing the obvious, I don't see that JSForce supports yet the recently released REST API batch function.

Doc link: http://docs.releasenotes.salesforce.com/en-us/summer15/release-notes/rn_api_rest.htm

**Comment by stomita** [comment]
Batch support is not included yet. By the way, you can use much lower-level call `Connection#requestPost()` and pass the batch request message.

``` javascript
conn.requestPost('/composite/batch', {
  "batchRequests" : [{
    "method" : "PATCH",
    "url" : "v34.0/sobjects/account/001D000000K0fXOIAZ",
    "richInput" : {"Name" : "NewName"}
  },{
    "method" : "GET",
    "url" : "v34.0/sobjects/account/001D000000K0fXOIAZ"
  }]
}, function(err, res) {
  // ...
}) 
```

In future release we may add more excellent support for batch API request sending.

**Comment by ArcusSimonCull** [comment]
Any plans to implement this yet? Considering we now have Composite, Batch and Tree requests?
